src/main/webapp/script.js

```diff
@@ -44,19 +45,22 @@ function renderSingleTask(task) {
  * reports if there are problems.
  */
 function updateView(results) {
-  // TODO: show how many tasks were scheduled, notify about conflicts/problems
+  alert(results);
```

Comment:
also ok for a demo but I would create a proper UI to show a message to the user.